src/services/transaction-service.js

```diff
@@ -0,0 +1,11 @@
+import { StorageService } from "./storage";
+
+export class TransactionListService {
+  get() {
```

Comment:
if you are using models --this service should resolve with array of `Transaction` models and you should be able to set a array of `Transaction`s

Reply:
I can't understand